lms/templates/courseware/course_about.html

```diff
@@ -62,7 +62,7 @@
                         'You may need to {start_logout_tag}log out{end_tag} then try the enroll button again. '
                         'Please visit the {start_help_tag}help page{end_tag} for a possible solution.')).format(
                           start_help_tag=HTML("<a href='{url}'>").format(url=marketing_link('FAQ')), end_tag=HTML('</a>'),
-                          start_logout_tag=HTML("<a href='{url}'>").format(url=reverse('auth_logout_redirect'))
+                          start_logout_tag=HTML("<a href='{url}'>").format(url=reverse('colaraz_features:auth_logout_redirect'_logout_redirect'))
```

Comment:
```suggestion
                          start_logout_tag=HTML("<a href='{url}'>").format(url=reverse('colaraz_features:auth_logout_redirect'))
```
I think it was left here by mistake

Reply:
Already update.